tools/clang/test/HLSL/workgraph/member_write_diagnostics.hlsl

```diff
@@ -110,7 +111,7 @@ void node07(NodeOutput<RECORD> output7)
 [Shader("node")]
 [NumThreads(1024,1,1)]
 [NodeLaunch("broadcasting")]
-[NodeMaxDispatchGrid(1024,1,1)]
+[NodeDispatchGrid(1024,1,1)]
```

Comment:
Were the changes to add `SV_DispatchGrid` to the RECORD struct and also the switching of `NodeMaxDispatchGrid` to `NodeDispatchGrid` both necessary? It seems like one or the other would have silenced the error as I understand it. 

Same for the next two test files

Reply:
Yes, they were. Without changing `NodeMaxDispatchGrid` to `NodeDispatchGrid` the node would have to have an extra argument of type `DispatchNodeInputRecord<RECORD>` added to make use of the `SV_DispatchGrid` in `RECORD`. Changing `NodeMaxDispatchGrid` to `NodeDispatchGrid` seems like an easier change and not affecting the test.

Reply:
got it! thanks!